src/rpc/blockchain.cpp

```diff
@@ -781,7 +781,8 @@ static RPCHelpMan getblockfrompeer()
         "getblockfrompeer",
         "\nAttempt to fetch block from a given peer.\n"
         "\nWe must have the header for this block, e.g. using submitheader.\n"
-        "\nReturns {} if a block-request was successfully scheduled\n",
+        "Subsequent calls for the same block and a new peer will cause the response from the previous peer to be ignored.\n"
+        "\nReturns an empty JSON object if a block-request was successfully scheduled.\n",
```

Comment:
```suggestion
        "\nReturns an empty JSON object if a block-request was successfully scheduled.",
```

;) after https://github.com/bitcoin/bitcoin/pull/23714

Reply:
Done, and rebased.